src/charts/vertical_bar.coffee

```diff
@@ -516,10 +516,15 @@ Ember.Charts.VerticalBarComponent = Ember.Charts.ChartComponent.extend(
       .tickSize(@get 'graphicWidth')
       .tickFormat(@get 'formatValueAxis')
 
+    gYAxis = @get('yAxis')
+
+    # find the correct size of graphicLeft in order to fit the Labels perfectly
+    @set 'graphicLeft', @maxLabelLength(gYAxis.selectAll('text')) + @get 'labelPadding'
```

Comment:
hmmm perhaps instead of maxLabelLength in no_margin_chart_mixin I could have graphicLeftCalculator or something. I guess I didn't do that because I couldn't think of a good name for graphicLeftCalculator (and still can't think of a good name for it =p )